app/src/main/java/com/example/paulsuarez/a26_simple_android_apps_simple_counter/MainActivity.java

```diff
@@ -0,0 +1,208 @@
+
+    private EditText mEditText;
+    private Button mManualUpdate;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_main);
+
+        mEditText = findViewById(R.id.manualEntry);
+        mManualUpdate = findViewById(R.id.manualUpdate);
+
+        Log.d("clickApp", "it's running!");
+
+        mClickMe = findViewById(R.id.clickMe);
+
+        mAButton = findViewById(R.id.aClickMe);
+        mBButton = findViewById(R.id.bClickMe);
+        mCButton = findViewById(R.id.cClickMe);
+
+
+        mClicks = findViewById(R.id.clicks);
+
+        mAText = findViewById(R.id.aClicks);
+        mBText = findViewById(R.id.bClicks);
+        mCText = findViewById(R.id.cClicks);
+
+        mManualUpdate.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View view) {
+                String text = mEditText.toString();
+                String text2 = mEditText.getText().toString();
+
+                totalClicks = Integer.parseInt(text2);
+                mClicks.setText("clicks: " + totalClicks);
+            }
+        });
+
+        mClickMe.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                totalClicks++;
+                mClicks.setText("clicks: " + totalClicks);
+
+            }
+        });
+
+        mAButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                aClicks++;
+                mAText.setText("Bitcoins Ordered: " + aClicks);
+
+            }
+        });
+
+
+        mBButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                bClicks++;
+                mBText.setText("Litecoins Ordered: " + bClicks);
+
+            }
+        });
+
+
+        mCButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                cClicks++;
+                mCText.setText("EncryptNodes Ordered: " + cClicks);
+
+            }
+        });
+
+
+
+
+        // testing
+        currentIndex = 0;
+
+        picture = findViewById(R.id.picture);
+        name = findViewById(R.id.name);
+        desc = findViewById(R.id.description);
+
+        prev = findViewById(R.id.prev);
+        next = findViewById(R.id.next);
+
+        showCoin();
+
+        prev.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                decrement();
+            }
+        });
+        next.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                increment();
+            }
+        });
+        //
+
+
+
+    }
+
+
+
+
+    // trying to make a single scrollable app:
+
```

Comment:
Generally member variables are defined at the top of the class, so anyone coming in later knows where to look, so moving these up to the top would be a good habit to get into.